src/yang/ietf-tcp.yang

```diff
@@ -82,7 +82,8 @@ module ietf-tcp {
       type boolean;
       default "false";
       description
-        "Enable support of TCP-Authentication Option (TCP-AO).";
+        "When set to true, TCP-Authentication Option (TCP-AO) is
+         enabled."
```

Comment:
You are still missing the ; at the end of this statement, which is leading to this error:

Validating yang/ietf-tcp@2022-01-18.yang
yang/ietf-tcp@2022-01-18.yang:87: error: unterminated statement definition for keyword "description", looking at }
yang/ietf-tcp@2022-01-18.yang failed pyang validation

Reply:
Sorry, I missed that one. It should now be fixed.